src/us/kbase/workspace/database/provenance/FundingReference.java

```diff
@@ -5,83 +5,69 @@
 import java.util.List;
 import java.util.ArrayList;
 import java.util.Optional;
-import us.kbase.workspace.database.Util;
 
 /**
- * A funding reference for a resource, including the funding body and the grant
- * awarded.
+ * A funding reference for a resource, including the funding body and the grant.
```

Comment:
Just so I understand this change - it seems like it's tightening the requirements for who can be a funder. With the old funderID and funderName, an individual could be a funder, but now only orgs are allowed. Similarly, `award` seems more general than `grant`, and so it seems some funding sources can no longer be represented

Reply:
Correct. This brings the CE schema terminology in closer alignment with schema.org, which will be more helpful when generating search engine indexable content for static narratives. Although sources like DataCite and Crossref use 'award' and 'grant' interchangeably, 'award' is commonly used to refer to prizes, which is not what we want to represent here.

Reply:
I'm a little worried that this makes things a bit too specific - the workspace has always been written with the philosophy that it's not a a KBase specific service and could work for all kinds of data. I guess it's easier to open things up later vs. tighten them down though, so for now...